hack/svc/multi-stream.py

```diff
@@ -0,0 +1,64 @@
+import json
+import os
+import shutil
+import time
+import re
+import getopt
+import sys
+from subprocess import Popen
+
+
+class strims_ffmpeg:
+    def __init__(self, hostAddress="0.0.0.0") -> None:
+        self.hostAddress = hostAddress
+
+    def startStream(self, streamLink: str, streamKey: str) -> str:

```

Comment:
```suggestion
    def ffmpeg_cmd(self, streamLink: str, streamKey: str) -> str:
```
doesn't actually start a stream, just builds the ffmpeg command